capstone/fitness/src/main/java/com/google/sps/util/DataHandler.java

```diff
@@ -13,11 +13,17 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 import com.google.appengine.api.datastore.FetchOptions;
+import java.util.HashSet;
+import java.util.Set;
 
 
-public class UserHelper {
+public class DataHandler {
 
   // Constants used for datastore.
+  // TODO(@gabrieldg) 
+  //  - make private and change to getters and setters.
+  //  - Figure out why the identifiers do not work.
```

Comment:
What's not working?

Reply:
That comment is from something that was not included in this PR. Forgot to add to ignore that as well.